R/utilities-simulation-results.R

```diff
@@ -83,6 +83,42 @@ getOutputValues <- function(simulationResults, quantitiesOrPaths, individualIds
   return(output)
 }
 
+#' @export
+getOutputValuesTLF <- function(simulationResults, quantitiesOrPaths, individualIds = NULL) {

```

Comment:
@All: This is just a first try to reorganize the way we export results for a population simulation

@abdullahhamadeh @pchelle 
Is this the type of format that would be expected in order to be able to  plug into the the TLF?

4 Columns: IndivdualId, Time, Path, value

![image](https://user-images.githubusercontent.com/1041237/66673376-6e054280-ec2e-11e9-88b8-a602594362a0.png)

Reply:
@Yuri05 FYI: what do you think?